internal/pkg/consensus/expected.go

```diff
@@ -173,7 +173,9 @@ func (c *Expected) validateMining(ctx context.Context,
 	parentWeight fbig.Int,
 	parentReceiptRoot cid.Cid) error {
 
-	powerTable := NewPowerTableView(c.state.StateView(parentStateRoot))
+	stateView := c.state.StateView(parentStateRoot)
+	sigValidator := NewSignatureValidator(stateView)
```

Comment:
nit: Ideally we would would use this abstraction everywhere we need this translation.  I believe it's needed in the mining worker and the storage and market connectors.

Reply:
Mining worker signs things, but doesn't validate signatures as far as I can tell (it assumes the pool provides only valid messages).

Reply:
Done for the module connector common (but with unfortunate type signature mismatch)